client/src/components/layout/PlanetShow.js

```diff
@@ -28,6 +52,8 @@ const PlanetShow = (props) => {
       <img src={planet.imageUrl} className="planet-image" alt="image of planet" />
       <h1>{planet.name}</h1>
       <h4>{planet.description}</h4>
+      <ReviewList reviews={planet.reviews} id={planet.id} />
+      <NewReviewForm postReview={postReview} />
```

Comment:
this form should only be rendered if there is a user logged in. We dont want random people making infinite reviews on our site